sdk/contentsafety/Azure.AI.ContentSafety/api/Azure.AI.ContentSafety.netstandard2.0.cs

```diff
@@ -70,7 +70,9 @@ public partial class ContentSafetyClient
         public virtual Azure.Response AnalyzeText(Azure.Core.RequestContent content, Azure.RequestContext context = null) { throw null; }
         public virtual System.Threading.Tasks.Task<Azure.Response<Azure.AI.ContentSafety.AnalyzeTextResult>> AnalyzeTextAsync(Azure.AI.ContentSafety.AnalyzeTextOptions body, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
         public virtual System.Threading.Tasks.Task<Azure.Response> AnalyzeTextAsync(Azure.Core.RequestContent content, Azure.RequestContext context = null) { throw null; }
+        public virtual Azure.Response<Azure.AI.ContentSafety.TextBlocklist> CreateOrUpdateTextBlocklist(string blocklistName, Azure.AI.ContentSafety.TextBlocklist resource, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
```

Comment:
`Create` operations should not take a model property as an input parameter - `blocklistName` should be passed to the model constructor to create it.  Please see https://github.com/annelo-msft/gists/blob/main/patch-models/samples.md#1-create-a-new-resource for details.

Reply:
`Update` operations should not take what is a model property as an input parameter -- `blocklistName` should come from the model property.  The model should be retrieved via a `Get` operation prior to sending it in an update operation, please see https://github.com/annelo-msft/gists/blob/main/patch-models/samples.md#2-update-a-top-level-property for details.

Reply:
I discussed this with @m-nash before at here https://github.com/Azure/autorest.csharp/issues/3640. At that time, he thought it is by design. Michael, do you want to take a look at it again?